### PR TITLE
Updated thesis entries and new publications

### PR DESCRIPTION
**Pull request by SundermannC**

### CODE REVIEW

File: literature.bib [hunk 1]
```diff
@@ -369,6 +347,44 @@ @inproceedings{BHK:SPLC23
...
+	title = {{Parameterizations for Approximate #SAT Solvers}},
+	school = UniUlm,
+	address = Germany,
+	type = Bachelor,
+	typo3Tags = {SupervisorCS, SupervisorTT},
+	month = OCT,
+	year = 2023
+}
+
+
```
Comment (tthuem) [inline literature.bib:389]: remove linebreak

File: literature.bib [hunk 2]
```diff
@@ -369,6 +347,44 @@ @inproceedings{BHK:SPLC23
...
+	title = {{Cutting Edge T-Wise Sampling With ddnnife}},
+	school = UniUlm,
+	address = Germany,
+	type = Bachelor,
+	typo3Tags = {SupervisorCS, SupervisorTT, FMCounting, PLSampling},
+	month = JAN,
+	year = 2023
+}
+
+
```
Comment (tthuem) [inline literature.bib:402]: linebreak

File: literature.bib [hunk 3]
```diff
@@ -369,6 +347,44 @@ @inproceedings{BHK:SPLC23
...
+@mastersthesis{Licha23,
+	comments = NotToBePublished,
+	author = {Licha, Lars},
+	title = {{Cutting Edge T-Wise Sampling With ddnnife}},
+	school = UniUlm,
+	address = Germany,
+	type = Bachelor,
+	typo3Tags = {SupervisorCS, SupervisorTT, FMCounting, PLSampling},
```
Comment (tthuem) [inline literature.bib:397]: check keyword for sampling, does not seem to be consistent

File: literature.bib [hunk 4]
```diff
@@ -105,6 +105,19 @@ @mastersthesis{Schiessle23
 	year = 2023
 }
 
+@mastersthesis{Riesland24,
+	author = {Riesland, Jonathan},
+	title = {{Optimizing T-Wise Sampling on d-DNNFs with Objective Functions}},
+	school = UniUlm,
+	address = Germany,
+	type = Bachelor,
+	typo3Tags = {SupervisorCS, SupervisorTT, Sampling},
```
Comment (tthuem) [inline literature.bib:114]: check keyword for sampling, does not seem to be consistent

File: literature.bib [hunk 5]
```diff
@@ -369,6 +347,44 @@ @inproceedings{BHK:SPLC23
...
+}
+
+@mastersthesis{Schmid23,
+	comments = NotToBePublished,
+	author = {Schmid, Clemens},
+	title = {{Parameterizations for Approximate #SAT Solvers}},
+	school = UniUlm,
+	address = Germany,
+	type = Bachelor,
+	typo3Tags = {SupervisorCS, SupervisorTT},
```
Comment (tthuem) [inline literature.bib:384]: FMCounting?

**Review by tthuem — CHANGES_REQUESTED** [review]

File: literature.bib [hunk 6]
```diff
@@ -215,43 +204,32 @@ @mastersthesis{Guething23
...
-	month = AUG,
-	year = 2022,
-	note = {To appear}
+@proceedings{MODEVAR24,
+	title = {{Sixth International Workshop on Languages for Modelling Variability (MODEVAR 2024)}}, 
+	author = {Galasso-Carbonnel, Jessie and Sundermann, Chico},
+	booktitle = MODEVAR,
+	institution = arXiv,
+	url = {https://arxiv.org/abs/2402.15511},
+	tt-tags = {Editorial},
```
[the remark this answers is not in the document]
Comment (pmbittner) [reply literature.bib:215]: The tt-tags are just for Thomas, and his variant of the MibTeX-Website. The tt-tags are ignored by the Typo3Exporter in MibTeX (and everything else).

As of now, the Typo3Exporter only checks for editorials by Thomas. It does so by looking whether Thomas' name appears in the `editor` field, and if so, adds a special Tag for Typo3 that our filters on the website can refer to. I can easily extend the Typo3Exporter to also listen for Chico as editor if required (e.g.,, if Chico wants to have his editorials as a special publication list on his website, as we have for Thomas).

**tl;dr**: For this to work, all we have to do in BibTags is: Add Chico Sundermann to the `editor` field.

[the remark this answers is not in the document]
Comment (tthuem) [reply literature.bib:215]: @pmbittner Whatever is easier for MibTeX and the Typo3Exporter is fine with me.

[the remark this answers is not in the document]
Comment (pmbittner) [reply literature.bib:215]: Ok.

@SundermannC Please do the following:
- Remove the tt-tags here, these are for Thomas' personal use and usually flag entries where _he_ is editor (not someone else). The tt-tags are irrelevant for Typo3.
- Change the `author` field to `editor`. We do this for all editorials (e.g., `MODEVARatSPLC21`).
- If you are interested in having editorials appear on your website, please get in touch with me.

File: literature.bib [hunk 7]
```diff
@@ -215,43 +204,32 @@ @mastersthesis{Guething23
...
 	title = {{On the Benefits of Knowledge Compilation for Feature-Model Analyses}},
 	journal = AMAI,
 	publisher = Springer,
 	typo3Tags = {FMCounting},
-	month = OCT,
+	month = NOV,
 	year = 2023,
-	note = {To appear}
+	doi = {10.1007/s10472-023-09906-6}
 }
```
[the remark this answers is not in the document]
Comment (SundermannC) [reply literature.bib:242]: Changed it to "To appear" again for now.
Also, changed year and removed month

File: literature.bib (hunk 3 above)
Comment (SundermannC) [reply literature.bib]: I changed the keyword to Sampling which seems to be more common, but PLSampling also appears in:
BKH+:VaMoS24
HSO+:VaMoS24
KHS+:VaMoS24
Is there a semantic difference between those? How should we proceed here?

Comment (tthuem) [reply literature.bib]: @skrieter
